### PR TITLE
docs: clarify 'Table.read_rows' snippet

### LINKED ISSUE #45: Documentation: Clarify read_rows
**Issue**
At first glance at the [provided code block](https://googleapis.dev/python/bigtable/latest/table.html#google.cloud.bigtable.table.Table.read_rows) in the docs it is unclear how to manipulate the `PartialRowsData`. I don't think this change is necessary at all, but it did provide a small blocking point while friction logging, so it may be helpful.

**Suggestion**
Provide a code block similar to that found in the [cloud.google docs](https://cloud.google.com/bigtable/docs/samples-python-hello#scanning_all_table_rows).

### PR DESCRIPTION
**Pull request by HemangChothani**
Fixes #45

### CODE REVIEW

File: docs/snippets_table.py [hunk 1]
```diff
@@ -203,6 +203,11 @@ def test_bigtable_write_read_drop_truncate():
     # Read full table
     partial_rows = table.read_rows()
     read_rows = [row for row in partial_rows]
```
Comment (tseaver) [inline docs/snippets_table.py:205]: No need to convert to a list, just iterate below, matching the [example in the API docs](https://cloud.google.com/bigtable/docs/samples-python-hello#scanning_all_table_rows).

File: docs/snippets_table.py [hunk 2]
```diff
@@ -203,6 +203,11 @@ def test_bigtable_write_read_drop_truncate():
...
+
+    # Read row's value
+    for row in read_rows:
```
Comment (tseaver) [inline docs/snippets_table.py:208]:
```suggestion
    for row in partial_rows:
```

File: docs/snippets_table.py [hunk 3]
```diff
@@ -202,9 +202,15 @@ def test_bigtable_write_read_drop_truncate():
...
     # Read full table
     partial_rows = table.read_rows()
-    read_rows = [row for row in partial_rows]
+
+    # Read row's value
+    total_rows = []
+    for row in partial_rows:
+        cell = row.cells[COLUMN_FAMILY_ID][col_name][0]
+        print(cell.value.decode("utf-8"))
+        total_rows.append(cell)
```
Comment (HemangChothani) [inline docs/snippets_table.py:211]: Append values into the list to assert with original row's values

File: docs/snippets_table.py [hunk 4]
```diff
@@ -202,14 +202,15 @@ def test_bigtable_write_read_drop_truncate():
...
     # Read row's value
-    for row in read_rows:
+    total_rows = []
+    for row in partial_rows:
         cell = row.cells[COLUMN_FAMILY_ID][col_name][0]
         print(cell.value.decode("utf-8"))
+        total_rows.append(cell)
     # [END bigtable_read_rows]
-    assert len(read_rows) == len(rows)
+    assert len(list(total_rows)) == len(rows)
```
Comment (tseaver) [inline docs/snippets_table.py:213]:
```suggestion
    assert len(total_rows) == len(rows)
```

**Pull request merged**